core/rpc-primitives/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
Maybe let's move this crate to `core/rpc/primitives` instead of `core/rpc-primitives`?

Reply:
keeping the same crate name? k!